src/discordBot.py

```diff
@@ -253,6 +253,28 @@ async def sendDailyRecap():
     recap = await dedicated_channel.send(embed=embed)
     await recap.pin()
 
+aiocron.crontab('59 7 * * *')
+async def send_top16_daily_recap():
+    top16_players_in_each_region = leaderboardBot.get_leaderboard_range(1,16)
+
+    discord_payload = ""
+
+    def append_to_discord_payload(string):
```

Comment:
discord_payload isn't defined in the scope of the function

Reply:
Ah right, because there is read scope here but not scope to write to that variable here.